mkinit/formatting.py

```diff
@@ -70,12 +71,11 @@ def _insert_autogen_text(modpath, initstr):
 
     QUICKFIX_REMOVE_LEADING_NEWLINES = 1
     if QUICKFIX_REMOVE_LEADING_NEWLINES:
-        initstr_ = initstr_.lstrip('\n')
+        initstr_ = initstr_.lstrip("\n")
 
     new_lines = lines[:startline] + [initstr_] + lines[endline:]
 
     new_text = "".join(new_lines).rstrip() + "\n"
-    print(new_text)
```

Comment:
This print never used to be here, and I wasn't sure why everything was being printed when verbose was disabled and `inplace` was set, so I removed it. Let me know if this was a mistake

Reply:
It's very possible that it's a mistake. I put print statements in for debugging all the time, and I often forget to remove them before committing.